sql/src/SqlCsv.sk

```diff
@@ -824,6 +825,7 @@ fun replayDiff(
 ): SKStore.ContextOp {
   SKDB.getReplicationState(context, Some(identity)) match {
   | Some(SKDB.RSError()) ->
+    SKDBMonitor.addReplicationWrites(1);
```

Comment:
It would be really valuable to break this out and measure explicitly # of reboots. This should be a rare event that we want to minimise.